integration-cli/docker_cli_images_test.go

```diff
@@ -193,9 +173,8 @@ func (s *DockerSuite) TestImagesEnsureDanglingImageOnlyListedOnce(c *check.C) {
 	dockerCmd(c, "tag", "-f", "busybox", "foobox")
 
 	out, _ = dockerCmd(c, "images", "-q", "-f", "dangling=true")
-	if e, a := 1, strings.Count(out, imageID); e != a {
-		c.Fatalf("expected 1 dangling image, got %d: %s", a, out)
-	}
+	a := strings.Count(out, imageID)
```

Comment:
Can we give this a better name, or skip the variable all-together and call `strings.Count` in the Assert?